addons/website_profile/views/website_profile.xml

```diff
@@ -363,11 +357,8 @@
                                         t-attf-src="/website_profile/static/src/img/badge_#{badge.badge_id.level}.svg"
                                         class="o_object_fit_cover me-0"
                                         t-att-alt="badge.badge_id.name"/>
-                                    <img t-else=""
-                                        width="38" height="38"
-                                        t-att-src="website.image_url(badge.badge_id, 'image_128')"
-                                        class="o_object_fit_cover me-0"
-                                        t-att-alt="badge.badge_id.name"/>
+                                    <div t-else="" t-field="badge.badge_id.image_1920"
```

Comment:
Could we switch those around?

Always better to see "I have an image, show it, otherwise use this fallback" than the other way around.